### DISCUSSION TITLE
Interactive Scenarios & SUMO Interface Error in tutorial : No module named ‘sumocr.maps

### DISCUSSION THREAD

**Discussion by marihansherif** [CommonRoad/commonroad-io#6] [Q&A] [answered]
I tried to run the first tutorial of interactive scenarios but I get this error 
![Screenshot 2024-06-09 002119](https://github.com/CommonRoad/commonroad-io/assets/73167635/bdff2cff-67a1-451a-80cb-fe7647fa9f27)
I am using windows 11 and I have successfully installed SUMO and I have set the SUMO_HOME and PATH and PYTHONPATH manually using windows settings

**Comment by florian-fi** [comment, marked as answer]
See https://commonroad.in.tum.de/forum/t/error-in-tutorial-no-module-named-sumocr-maps/1565
